build.gradle

```diff
@@ -45,27 +57,33 @@ subprojects {
     targetCompatibility = 1.7;
 
     repositories {
-        mavenCentral()
+        //mavenCentral()
         maven {
             url = "$MAVEN_REPOSITORY_URL"
         }
+        maven {
+            url = "$MAVEN_SNAPSHOT_REPOSITORY_URL"
+        }
         mavenLocal()
     }
 
     dependencies {
-
-        // TODO:2015-11-19:alexander.litus: change Spine local dependency to Maven central repository dependency
         // when Spine can be used for evaluation in real projects.
+        // Spine
+        compile group: 'org.spine3', name: 'server', version: project.SPINE_VERSION, changing: true
+        compile group: 'org.spine3', name: 'client', version: project.SPINE_VERSION, changing: true
+        compile group: 'org.spine3', name: 'testutil', version: project.SPINE_VERSION, changing: true
+        compile group: 'org.spine3', name: 'examples', version: "0.2", changing: true
 
         // Guava
-        compile group: 'com.google.guava', name: 'guava', version: '19.0-rc2'
+        compile group: 'com.google.guava', name: 'guava', version: '19.0'
 
         // Findbugs
         compile group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.0'
 
         // SLF4J
-        compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.12'
-        compile group: 'org.slf4j', name: 'slf4j-jdk14', version: '1.7.12'
+        compile group: 'org.slf4j', name: 'slf4j-api', version: '1.7.21'
```

Comment:
As long as you modify it, could you please extract the repeated versions as constants?